docs/book/src/sway-program-types/libraries.md

```diff
@@ -86,20 +85,102 @@ The `dep` keyword in the main library includes a dependency on another library,
 
 ## Using Libraries
 
-Libraries can be imported using the `use` keyword and with a `::` separating the name of the library and the import.
+There are two types of Sway libraries, based on their location and how they can be imported.
 
-Here is an example of importing the `get<T>` and `store<T>` functions from the `storage` library.
+### Internal Libraries
 
-```sway
-use std::storage::{get, store};
+Internal libraries are located within the project's `src` directory alongside `main.sw` as shown below:
+
+```bash
+$ tree
+.
+├── Cargo.toml
+├── Forc.toml
+└── src
+    ├── internal_lib.sw
+    └── main.sw
 ```
 
-Wildcard imports using `*` are supported, but it is always recommended to use explicit imports where possible.
 
-Libraries _other than the standard library_ have to be added as a dependency in `Forc.toml`. This can be done by adding a path to the library in the `[dependencies]` section. For example:
+As `internal_lib` is an internal library, it can be imported into `main.sw` as follows:
+- Use the `dep` keyword followed by the library name to make the internal library a dependancy
+- Use the `use` keyword with a `::` separating the name of the library and the imported item(s)
+
+```sway
+dep internal_lib; // Assuming the library name in `internal_lib.sw` is `internal_lib`
+
+use internal_lib::mint;
+
+`mint` from `internal_library` is now available in this file
```

Comment:
```suggestion
// `mint` from `internal_library` is now available in this file
```

Reply:
Oh, wait yes! Sorry, let me set up the lint checker.